google-cloud-spanner-hibernate-performance-testing/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>google-cloud-spanner-hibernate</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>0.2.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Google Cloud Spanner Hibernate Performance Testing</name>
+  <artifactId>google-cloud-spanner-hibernate-performance-testing</artifactId>
+
+  <properties>
+    <log4j.version>1.2.17</log4j.version>
+    <google-cloud-spanner.version>1.43.0</google-cloud-spanner.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-spanner-hibernate-dialect</artifactId>
+      <version>${project.version}</version>
```

Comment:
Please use dependency-management vs. setting versions in `<dependencies>`.

Reply:
Done.